examples/surface_texture_and_colors.py

```diff
@@ -0,0 +1,56 @@
+"""
+Surface with texture and vertex_colors
+==========
+
+Display a 3D surface with texture mapping and colors
+
+.. tags:: visualization-nD
+"""
+import numpy as np
+from vispy.io import imread, load_data_file, read_mesh
+
+import napari
+
+# create the viewer and window
+viewer = napari.Viewer(ndisplay=3)
+
+# load the model and texture
+mesh_path = load_data_file('spot/spot.obj.gz')
+vertices, faces, normals, texcoords = read_mesh(mesh_path)
+n = len(vertices)
+texture_path = load_data_file('spot/spot.png')
+texture = np.flipud(imread(texture_path))
+
+offset = np.zeros(vertices.shape)
+offset[:, 0] = 1
+
+viewer.add_surface(
+    (vertices, faces, np.random.random((3, 3, n))),
```

Comment:
You use random, do you want to set the seed at be beginning of the example to get reproducible results ?